external/flume/src/test/scala/org/apache/spark/streaming/flume/FlumeStreamSuite.scala

```diff
@@ -76,7 +75,7 @@ class FlumeStreamSuite extends FunSuite with BeforeAndAfter with Matchers with L
 
   /** Find a free port */
   private def findFreePort(): Int = {
-    Utils.startServiceOnPort(23456, (trialPort: Int) => {
+    Utils.startServiceOnPort((math.random * Int.MaxValue).toInt, (trialPort: Int) => {
```

Comment:
I like the idea but surely you don't mean to choose a random port from 0 to 2 billion? surely something in [1024,65536)? `1024 + new Random().nextInt(65536 - 1024)` or similar?